File: mySort.java
import java.util.Arrays;
import java.util.Scanner;

public class mySort {
    // The following is the main, containing the array.
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        int [] Myarray = {1,2,5,6,3,4,77,8,9,11,10,-8};
        mySort(Myarray);
    }
    // The following is the method, to sort an array
    public static void mySort(int [] Myarray) {
        int temp = 0;
        for (int i = 0; i < Myarray.length - 1; i++) {
            if (Myarray[i] > Myarray[i + 1]) {
                temp = Myarray[i];
                Myarray[i] = Myarray[i + 1];
                Myarray[i + 1] = temp;
                i =  0;
            }
        }
        System.out.print(Arrays.toString(Myarray));
    }
    }

File: Mysort.py
def sort(Myarray):
    temp = 0
    u = -1
    while u <= len(Myarray):
        u += 1
        for i in range(0, len(Myarray) - 1):
            if Myarray[i] > Myarray[i + 1]:
                temp = Myarray[i]
                Myarray[i] = Myarray[i + 1]
                Myarray[i + 1] = temp


    return Myarray
# this is an example of the function to sort the array; 1,2,5,6,3,4 etc
Myarray = [1,2,5,6,3,4,77,8,9,11,10,-8]
print(sort(Myarray))





























